editor/editor_node.cpp

```diff
@@ -7079,127 +6501,88 @@ EditorNode::EditorNode() {
 	title_bar = memnew(EditorTitleBar);
 	main_vbox->add_child(title_bar);
 
-	left_l_hsplit = memnew(HSplitContainer);
+	left_l_hsplit = memnew(DockSplitContainer);
+	left_l_hsplit->set_name("left_l_hsplit");
 	main_vbox->add_child(left_l_hsplit);
 
 	left_l_hsplit->set_v_size_flags(Control::SIZE_EXPAND_FILL);
 
-	left_l_vsplit = memnew(VSplitContainer);
+	left_l_vsplit = memnew(DockSplitContainer);
+	left_l_vsplit->set_name("left_l_vsplit");
+	left_l_vsplit->set_vertical(true);
```

Comment:
```suggestion
	left_l_vsplit = memnew(DockSplitContainer(true));
	left_l_vsplit->set_name("left_l_vsplit");
```

You can make it vertical via the constructor, can you not? It should be inherited from the regular SplitContainer, especially when you remove the empty constructor.

Reply:
I'll need to make a new constructor for this, it's not automatically inheriting the SplitContainer constructor, even if I remove the empty one.

Reply:
Well, then keep it as is. Verbosity is not bad in this case. (And I need to refresh my knowledge on how constructors in C++ work :D)